modules/constructor.js

```diff
@@ -72,34 +57,42 @@ class UI {
       }
       const btnId = e.target.dataset.id;
       // remove from array.
-      UI.removeArrayTodo(btnId);
+      Todo.removeArrayTodo(btnId);
     });
   }
 
   static removeArrayTodo(id) {
     todoArr = todoArr.filter((item) => item.id !== +id);
     Storage.addTodStorage(todoArr);
-    UI.colorChanger();
+    if (todoArr.length === 0) {
+      Todo.displayNothing();
+    }
+    Todo.colorChanger();
   }
 }
 
-// form part
 form.addEventListener('submit', (e) => {
   e.preventDefault();
   const id = Math.random() * 1000000;
   const todo = new Todo(id, bookTitle.value, bookAuthor.value);
   todoArr = [...todoArr, todo];
-  UI.displayData();
-  UI.colorChanger();
-  UI.clearInput();
+  Todo.displayData();
+  Todo.colorChanger();
+  Todo.clearInput();
   // add to storage
   Storage.addTodStorage(todoArr);
+  console.log(Storage.addTodStorage(todoArr));
 });
```

Comment:
- [ ] The form is not submitted when I try submitting data. Add and remove features are expected to work fine. Kindly see that the form captures data and submits it and the data is rendered on the list page. Also, please make sure that the delete feature works fine as expected.